### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ilona-Gogol**
- [DEMO LINK](https://Ilona-Gogol.github.io/js_task_generate_table_DOM/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your solution

**Comment by Ilona-Gogol** [comment]
-  [DEMO LINK](https://Ilona-Gogol.github.io/js_task_generate_table_DOM/)

File: src/scripts/main.js [hunk 1]
```diff
@@ -354,7 +354,28 @@ const people = [
...
+  const newRow = table.insertRow();
+  const nameCell = newRow.insertCell(0);
+  const genderCell = newRow.insertCell(1);
+  const bornCell = newRow.insertCell(2);
+  const diedCell = newRow.insertCell(3);
+  const ageCell = newRow.insertCell(4);
+  const centuryCell = newRow.insertCell(5);
+
+  nameCell.textContent = person.name;
+  genderCell.textContent = person.gender;
```
Comment (anastasiiavorobiova) [inline src/scripts/main.js]:
```suggestion
  const MAN = 'm';
  
  genderCell.textContent = person.sex === MAN ? 'Male' : 'Female';
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Screenshot 2024-10-09 at 17 36 54
Great start! Consider fixing gender column

**Pull request closed without merging**